stream/src/main/scala/org/apache/pekko/stream/scaladsl/Flow.scala

```diff
@@ -2521,6 +2526,11 @@ trait FlowOps[+Out, +Mat] {
    */
   def flatMapConcat[T, M](f: Out => Graph[SourceShape[T], M]): Repr[T] = map(f).via(new FlattenMerge[T, M](1))
 
+  /**
```

Comment:
Can we add `@since` to new methods. This is required on all new methods.

Can we also annotate the new methods as experimental? It gives us some fallback if we find that users using for comprehensions are having trouble - and that we need to make changes to help them.